pkg/create/templates/scripts.go

```diff
@@ -20,11 +20,11 @@ fi
 #
 if [ "$(ls /tmp/artifacts/ 2>/dev/null)" ]; then
   echo "---> Restoring build artifacts..."
-  mv /tmp/artifacts/. ./
+  mv /tmp/artifacts/* ./
 fi
 
 echo "---> Installing application source..."
-cp -Rf /tmp/src/. ./
+cp -Rf /tmp/src/* ./
```

Comment:
this change is going to miss hidden files.  you need to set shopt dotglob if you want to make this change.